xlators/mgmt/glusterd/src/glusterd.c

```diff
@@ -451,7 +448,6 @@ glusterd_rpcsvc_options_build(dict_t *options)
     int ret = 0;
     uint32_t backlog = 0;
     xlator_t *this = THIS;
-    GF_ASSERT(this);
```

Comment:
nit: 'this' is only used once in this function, in an error log message. Just delete this variable and call the gf_msg() with 'THIS->name' instead.

Reply:
Ack.